src/numberOperations.cpp

```diff
@@ -0,0 +1,45 @@
+// numberOperation namespace functions
+#include "folf.hpp"
+#include "omp.h"
+#include <cmath>
+#include <random>
+#include <chrono>
+
+bool folf::numberOperations::checkPrime(const unsigned long long *num) {
+    // taking the square root to speed up the loop
+    // placing the result into another variable to save more time (seems like c++ calculates the sqrt on every loop if its inside the case)
```

Comment:
Y no multiline comment?

Reply:
because its cleaner if i just make two normal commends:

// taking the square root to speed up the loop
// placing the result into another variable to save more time (seems like c++ calculates the sqrt on every loop if its inside the case)

instead of:
/* taking the square root to speed up the loop
placing the result into another variable to save more time (seems like c++ calculates the sqrt on every loop if its inside the case) /*

or 
/* 
taking the square root to speed up the loop
placing the result into another variable to save more time (seems like c++ calculates the sqrt on every loop if its inside the case) 
/*

Reply:
So those two comments don't belong to each other and to two different lines? If they do and are both part of one/the same explanation, it'd be better to use a multiline comment. Otherwise not. (Why I asked)

Reply:
will move the comment up, that are two separate comments